config/locales/client.en.yml

```diff
@@ -2271,6 +2271,7 @@ en:
         body: "Right now this message is only being sent to yourself!"
       slow_mode:
         error: "This topic is in slow mode. You already posted recently; you can post again in %{timeLeft}."
+      user_not_seen_title: "User hasn't been seen in a long time"
```

Comment:
This should have a plural option too, like the description.

Reply:
Sure, done.